indico/modules/events/sessions/blueprint.py

```diff
@@ -41,3 +42,6 @@
 _bp.add_url_rule('/manage/sessions/coordinator-privs/', 'coordinator_privs_rest', RHSessionCoordinatorPrivs)
 _bp.add_url_rule('/manage/sessions/coordinator-privs/<priv>', 'coordinator_privs_rest', RHSessionCoordinatorPrivs,
                  methods=('PUT', 'DELETE'))
+
+# Display
+_bp.add_url_rule('/sessions/', 'display_session_list', RHDisplaySessionList)
```

Comment:
Not a good URL for something specific to your sessions. That URL would be used for a session list.

Maybe `/sessions/mine` or `/my-sessions`?